### ISSUE TITLE
Upgrade Spring Boot to the latest version

### ISSUE THREAD

**Issue by robertmcnees** [spring-guides/gs-rest-service#149]
Update the guide to use the most recent Spring Boot version.

Files that require changes are:
```
initial/build.gradle
initial/pom.xml
complete/build.gradle
complete/pom.xml
```

**Comment by bodote** [comment]
I did make a pull request for upgrading this repo. 
But I am wondering, why not using dependapod or renovate for this ?

**Comment by robertmcnees** [comment]
Hi @bodote .  Thanks for the comment and the PR!  I merged the PR.

I agree there are more automated ways to keep our dependencies up to date.  For this project I want to keep the effort manual so we can provide issues for first time contributors.  We've found a large audience that would like to contribute to OSS but aren't sure how to navigate the process.  We're using this issue (and others in the spring-guides org) largely for educational purposes.  Depending on the interest we may look at using an automated workflow in the future.

I will close this issue as the PR is merged.  There will likely be new opportunities to upgrade the Boot version in the future.
